modules/apps/document-library/document-library-video/src/main/java/com/liferay/document/library/video/internal/display/context/DLVideoExternalShortcutDLEditFileEntryDisplayContext.java

```diff
@@ -62,6 +62,24 @@ public DLFilePicker getDLFilePicker(String onFilePickCallback) {
 			_dlVideoExternalShortcut, onFilePickCallback, _servletContext);
 	}
 
+	@Override
+	public long getInheritableParentFolderId() {
+		if (parentDisplayContext == null) {
+			return -1;
+		}
+
+		return parentDisplayContext.getInheritableParentFolderId();
+	}
+
+	@Override
+	public String getInheritableParentFolderName() {
+		if (parentDisplayContext == null) {
+			return null;
+		}
+
+		return parentDisplayContext.getInheritableParentFolderName();
+	}
+
```

Comment:
I think these codes should be added in BaseDLEditFileEntryDisplayContext instead. And looks like we don't need to check null here. Thanks.